config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml

```diff
@@ -10,22 +10,9 @@ presubmits:
       description: "make sure each PR to kernel-module-management has a single commit."
     spec:
       containers:
-      - image: ghcr.io/supportpal/github-gh-cli
+      - image: bitnami/git:latest
```

Comment:
How about using the [Kubernetes image](https://github.com/kubernetes/test-infra/tree/master/images/git)?

Reply:
Fixed.